.github/workflows/cd.yml

```diff
@@ -0,0 +1,82 @@
+name: Release
+
+on:
+  push:
+    tags:
+      - "*"
+
+jobs:
+  calculate-version:
+    name: Calculate Version
+    runs-on: ubuntu-latest
```

Comment:
Is there a reason to run this on Linux and not do it all on Windows? I imagine it would be faster to do it as a single job.

Reply:
Tends to be convention to use linux by default and only use a Windows image for things that specifically require Windows, such as WPF solutions which have no Linux compatibility. Will look into making it a single job.